### ISSUE TITLE
Auto resolution policy in composition specification

### ISSUE THREAD

**Issue by ralvarep** [fp7-netide/Engine#112]
Hi @schwabe, I continue testing the Java Core with the different modes of resolution policy in the scenario that I mentioned at #110: 

Alice (00:00:00:00:00:e1) <==> SW <==> Bob (00:00:00:00:00:e2)
using OF10 and with the apps of switch and firewall.

The resolution policy modes of _priority_, _ignore_ and _pass_ are fine, but when I use _auto_ mode, the core works in the same way as _ignore_ mode. It doesn't configure any flow_mod from the apps.

The composition file is the following:

```
<?xml version="1.0" ?>
<CompositionSpecification
        xmlns="http://netide.eu/schemas/compositionspecification/v1">
    <Modules>
        <Module id="Firewall" loaderIdentification="firewall.py" noFenceSupport="false">
            <CallCondition events="packetIn" datapaths="1"/>
        </Module>
        <Module id="SimpleSwitch" loaderIdentification="simple_switch.py" noFenceSupport="false">
            <CallCondition events="packetIn" datapaths="1"/>
        </Module>
    </Modules>
    <Composition>
        <ParallelCall resolutionPolicy="auto">
            <ModuleCall module="SimpleSwitch"/>
            <ModuleCall module="Firewall"/>
        </ParallelCall>
    </Composition>
</CompositionSpecification>
```

Is this behaviour ok? How does _auto_ mode work?

Thanks for your help @schwabe.

**Comment by schwabe** [comment]
auto is only implemented for 1.0 and only for a small subset of fields since it uses the same classes as the serial composition.

**Comment by ralvarep** [comment]
Thanks @schwabe, I close this issue.
